Source/AwsCommonRuntimeKit/auth/credentials/CRTCredentialsProvider.swift

```diff
@@ -48,14 +48,11 @@ class WrappedCRTCredentialsProvider {
 
         })
         let shutDownOptions = Self.setUpShutDownOptions(shutDownOptions: shutDownOptions)
-        let intPointer = UnsafeMutablePointer<Int>.allocate(capacity: 1)
-        intPointer.pointee = 1
-        let atomicVar = aws_atomic_var(value: UnsafeMutableRawPointer(intPointer))
+        let intPointer: UnsafeMutableRawPointer = fromPointer(ptr: 1)
```

Comment:
I hadn't noticed this code before. Does this imply that Swift doesn't have atomics?!

Reply:
updated this to use our own Atomic that creates the c type